cli/src/torii/routing.rs

```diff
@@ -110,14 +113,26 @@ pub(crate) async fn handle_instructions<W: WorldTrait>(
 }
 
 #[iroha_futures::telemetry_future]
+#[allow(unsafe_code, clippy::unimplemented)]
 pub(crate) async fn handle_queries<W: WorldTrait>(
     wsv: Arc<WorldStateView<W>>,
-    query_validator: Arc<IsQueryAllowedBoxed<W>>,
+    query_validator: Arc<IsQueryAllowedBoxed>,
     pagination: Pagination,
     request: VerifiedQueryRequest,
 ) -> Result<Scale<VersionedPaginatedQueryResult>, warp::http::Response<warp::hyper::Body>> {
+    // TODO: Very dirty, need to do something with it
```

Comment:
Need a ticket. We also need to understand both why this was necessary, and what needs to be done better.

Reply:
Done: #2292